MSVC 14.37の仕様変更への対応

## 不具合の内容

MSVC 14.37において仕様変更が行われたらしくそのままpyopenjtalkをインストールをすると`cl.exe`がexit code2でエラーを出しインストールができません。

アップデートによって影響範囲は広がっていくと考えられるため修正が必要です。
### 現象・ログ

理由としてはpyopenjtalkの`lib\open_jtalk\src\mecab/src\dictionary.cpp`で使われている（ここでは`src\mecab/src\dictionary.cpp`）`std::binary_function`が`functional`に含まれる？ように変更されるようになり、それがincludeされていないからだと思われます。これは[コミュニティ](https://learn.microsoft.com/en-us/answers/questions/1348183/std-binary-function-is-missing-in-msvc-14-37-32822)でも上がっています。

<details><summary>失敗したときのログ</summary>

```log
Using pip 23.2.1 from C:\Users\mymc\Desktop\python\learn_ai\.venv\lib\site-packages\pip (python 3.9)
Collecting pyopenjtalk
  Using cached pyopenjtalk-0.3.2.tar.gz (1.4 MB)
  Running command pip subprocess to install build dependencies
  Ignoring oldest-supported-numpy: markers 'python_version < "3.9"' don't match your environment
  Collecting setuptools<v60.0
    Using cached setuptools-59.8.0-py3-none-any.whl (952 kB)
  Collecting cython<3.0.0,>=0.28.0
    Obtaining dependency information for cython<3.0.0,>=0.28.0 from https://files.pythonhosted.org/packages/3f/d6/9eed523aeaca42acbaa3e6d3850edae780dc7f8da9df1bf6a2ceb851839c/Cython-0.29.36-py2.py3-none-any.whl.metadata
    Using cached Cython-0.29.36-py2.py3-none-any.whl.metadata (3.1 kB)
  Collecting cmake
    Obtaining dependency information for cmake from https://files.pythonhosted.org/packages/d4/73/ea47cc01e712ab804f1e40840ee8c5243cfdf73d34497cce27fa5d66a82d/cmake-3.27.2-py2.py3-none-win_amd64.whl.metadata
    Using cached cmake-3.27.2-py2.py3-none-win_amd64.whl.metadata (6.8 kB)
  Collecting numpy>=1.25.0
    Obtaining dependency information for numpy>=1.25.0 from https://files.pythonhosted.org/packages/df/18/181fb40f03090c6fbd061bb8b1f4c32453f7c602b0dc7c08b307baca7cd7/numpy-1.25.2-cp39-cp39-win_amd64.whl.metadata
    Using cached numpy-1.25.2-cp39-cp39-win_amd64.whl.metadata (5.7 kB)
  Using cached Cython-0.29.36-py2.py3-none-any.whl (988 kB)
  Using cached cmake-3.27.2-py2.py3-none-win_amd64.whl (34.6 MB)
  Using cached numpy-1.25.2-cp39-cp39-win_amd64.whl (15.6 MB)
  Installing collected packages: cmake, setuptools, numpy, cython
  Successfully installed cmake-3.27.2 cython-0.29.36 numpy-1.25.2 setuptools-59.8.0
  Installing build dependencies ... done
  Running command Getting requirements to build wheel
  setup.py:26: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    _CYTHON_INSTALLED = ver >= LooseVersion(min_cython_ver)
  -- Building for: Visual Studio 17 2022
  CMake Deprecation Warning at CMakeLists.txt:5 (cmake_minimum_required):
    Compatibility with CMake < 3.5 will be removed from a future version of
    CMake.

    Update the VERSION argument <min> value or use a ...<max> suffix to tell
    CMake that the project does not need compatibility with older versions.


  -- The C compiler identification is MSVC 19.37.32822.0
  -- The CXX compiler identification is MSVC 19.37.32822.0
  -- Detecting C compiler ABI info
  -- Detecting C compiler ABI info - done
  -- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2022/BuildTools/VC/Tools/MSVC/14.37.32822/bin/Hostx64/x64/cl.exe - skipped
  -- Detecting C compile features
  -- Detecting C compile features - done
  -- Detecting CXX compiler ABI info
  -- Detecting CXX compiler ABI info - done
  -- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2022/BuildTools/VC/Tools/MSVC/14.37.32822/bin/Hostx64/x64/cl.exe - skipped
  -- Detecting CXX compile features
  -- Detecting CXX compile features - done
  -- Looking for include file ctype.h
  -- Looking for include file ctype.h - found
  -- Looking for include file dirent.h
  -- Looking for include file dirent.h - not found
  -- Looking for include file fcntl.h
  -- Looking for include file fcntl.h - found
  -- Looking for include file inttypes.h
  -- Looking for include file inttypes.h - found
  -- Looking for include file io.h
  -- Looking for include file io.h - found
  -- Looking for include file memory.h
  -- Looking for include file memory.h - found
  -- Looking for include file setjmp.h
  -- Looking for include file setjmp.h - found
  -- Looking for include file stdint.h
  -- Looking for include file stdint.h - found
  -- Looking for include file stdlib.h
  -- Looking for include file stdlib.h - found
  -- Looking for include file strings.h
  -- Looking for include file strings.h - not found
  -- Looking for include file string.h
  -- Looking for include file string.h - found
  -- Looking for include file sys/mman.h
  -- Looking for include file sys/mman.h - not found
  -- Looking for include file sys/stat.h
  -- Looking for include file sys/stat.h - found
  -- Looking for include file sys/times.h
  -- Looking for include file sys/times.h - not found
  -- Looking for include file sys/types.h
  -- Looking for include file sys/types.h - found
  -- Looking for include file unistd.h
  -- Looking for include file unistd.h - not found
  -- Looking for include file windows.h
  -- Looking for include file windows.h - found
  -- Looking for getenv
  -- Looking for getenv - found
  -- Looking for getpagesize
  -- Looking for getpagesize - not found
  -- Looking for mmap
  -- Looking for mmap - not found
  -- Looking for opendir
  -- Looking for opendir - not found
  -- Looking for setjmp
  -- Looking for setjmp - found
  -- Looking for sqrt
  -- Looking for sqrt - found
  -- Looking for strstr
  -- Looking for strstr - found
  -- Looking for iconv_open in iconv
  -- Looking for iconv_open in iconv - not found
  -- Looking for sqrt in m
  -- Looking for sqrt in m - not found
  -- Looking for stddef.h
  -- Looking for stddef.h - found
  -- Check size of char
  -- Check size of char - done
  -- Check size of int
  -- Check size of int - done
  -- Check size of long
  -- Check size of long - done
  -- Check size of long long
  -- Check size of long long - done
  -- Check size of short
  -- Check size of short - done
  -- Check size of size_t
  -- Check size of size_t - done
  -- Configuring done (18.6s)
  CMake Warning (dev) at CMakeLists.txt:34 (add_library):
    Policy CMP0008 is not set: Libraries linked by full-path must have a valid
    library file name.  Run "cmake --help-policy CMP0008" for policy details.
    Use the cmake_policy command to set the policy and suppress this warning.

    Target "openjtalk" links to item

      C:/Users/mymc/AppData/Local/Temp/pip-install-w5__eo7i/pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc/lib/open_jtalk/src/build/dummy

    which is a full-path but not a valid library file name.
  This warning is for project developers.  Use -Wno-dev to suppress it.

  -- Generating done (0.0s)
  -- Build files have been written to: C:/Users/mymc/AppData/Local/Temp/pip-install-w5__eo7i/pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc/lib/open_jtalk/src/build
  fatal: not a git repository (or any of the parent directories): .git
  CMake is in the system path. Version:               cmake version 3.27.2

  CMake suite maintained and supported by Kitware (kitware.com/cmake).
  running egg_info
  writing pyopenjtalk.egg-info\PKG-INFO
  writing dependency_links to pyopenjtalk.egg-info\dependency_links.txt
  writing requirements to pyopenjtalk.egg-info\requires.txt
  writing top-level names to pyopenjtalk.egg-info\top_level.txt
  reading manifest file 'pyopenjtalk.egg-info\SOURCES.txt'
  reading manifest template 'MANIFEST.in'
  warning: no files found matching '*.cpp' under directory 'pyopenjtalk'
  warning: no files found matching 'LICENSE' under directory 'lib'
  adding license file 'LICENSE.md'
  writing manifest file 'pyopenjtalk.egg-info\SOURCES.txt'
  Getting requirements to build wheel ... done
  Running command pip subprocess to install backend dependencies
  Collecting wheel
    Obtaining dependency information for wheel from https://files.pythonhosted.org/packages/b8/8b/31273bf66016be6ad22bb7345c37ff350276cfd46e389a0c2ac5da9d9073/wheel-0.41.2-py3-none-any.whl.metadata
    Using cached wheel-0.41.2-py3-none-any.whl.metadata (2.2 kB)
  Using cached wheel-0.41.2-py3-none-any.whl (64 kB)
  Installing collected packages: wheel
  Successfully installed wheel-0.41.2
  Installing backend dependencies ... done
  Running command Preparing metadata (pyproject.toml)
  setup.py:26: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    _CYTHON_INSTALLED = ver >= LooseVersion(min_cython_ver)
  fatal: not a git repository (or any of the parent directories): .git
  CMake is in the system path. Version:               cmake version 3.27.2

  CMake suite maintained and supported by Kitware (kitware.com/cmake).
  running dist_info
  creating C:\Users\mymc\AppData\Local\Temp\pip-modern-metadata-0yjuq4d4\pyopenjtalk.egg-info
  writing C:\Users\mymc\AppData\Local\Temp\pip-modern-metadata-0yjuq4d4\pyopenjtalk.egg-info\PKG-INFO
  writing dependency_links to C:\Users\mymc\AppData\Local\Temp\pip-modern-metadata-0yjuq4d4\pyopenjtalk.egg-info\dependency_links.txt
  writing requirements to C:\Users\mymc\AppData\Local\Temp\pip-modern-metadata-0yjuq4d4\pyopenjtalk.egg-info\requires.txt
  writing top-level names to C:\Users\mymc\AppData\Local\Temp\pip-modern-metadata-0yjuq4d4\pyopenjtalk.egg-info\top_level.txt
  writing manifest file 'C:\Users\mymc\AppData\Local\Temp\pip-modern-metadata-0yjuq4d4\pyopenjtalk.egg-info\SOURCES.txt'
  reading manifest file 'C:\Users\mymc\AppData\Local\Temp\pip-modern-metadata-0yjuq4d4\pyopenjtalk.egg-info\SOURCES.txt'
  reading manifest template 'MANIFEST.in'
  warning: no files found matching '*.cpp' under directory 'pyopenjtalk'
  warning: no files found matching 'LICENSE' under directory 'lib'
  adding license file 'LICENSE.md'
  writing manifest file 'C:\Users\mymc\AppData\Local\Temp\pip-modern-metadata-0yjuq4d4\pyopenjtalk.egg-info\SOURCES.txt'
  creating 'C:\Users\mymc\AppData\Local\Temp\pip-modern-metadata-0yjuq4d4\pyopenjtalk.dist-info'
  Preparing metadata (pyproject.toml) ... done
Collecting numpy>=1.20.0 (from pyopenjtalk)
  Obtaining dependency information for numpy>=1.20.0 from https://files.pythonhosted.org/packages/df/18/181fb40f03090c6fbd061bb8b1f4c32453f7c602b0dc7c08b307baca7cd7/numpy-1.25.2-cp39-cp39-win_amd64.whl.metadata
  Using cached numpy-1.25.2-cp39-cp39-win_amd64.whl.metadata (5.7 kB)
Collecting six (from pyopenjtalk)
  Using cached six-1.16.0-py2.py3-none-any.whl (11 kB)
Collecting tqdm (from pyopenjtalk)
  Obtaining dependency information for tqdm from https://files.pythonhosted.org/packages/00/e5/f12a80907d0884e6dff9c16d0c0114d81b8cd07dc3ae54c5e962cc83037e/tqdm-4.66.1-py3-none-any.whl.metadata    
  Using cached tqdm-4.66.1-py3-none-any.whl.metadata (57 kB)
Collecting colorama (from tqdm->pyopenjtalk)
  Using cached colorama-0.4.6-py2.py3-none-any.whl (25 kB)
Using cached numpy-1.25.2-cp39-cp39-win_amd64.whl (15.6 MB)
Using cached tqdm-4.66.1-py3-none-any.whl (78 kB)
Building wheels for collected packages: pyopenjtalk
  Running command Building wheel for pyopenjtalk (pyproject.toml)
  setup.py:26: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
    _CYTHON_INSTALLED = ver >= LooseVersion(min_cython_ver)
  fatal: not a git repository (or any of the parent directories): .git
  CMake is in the system path. Version:               cmake version 3.27.2

  CMake suite maintained and supported by Kitware (kitware.com/cmake).
  running bdist_wheel
  running build
  running build_py
  -- Building version 0.3.2
  creating build\lib.win-amd64-3.9
  creating build\lib.win-amd64-3.9\pyopenjtalk
  copying pyopenjtalk\utils.py -> build\lib.win-amd64-3.9\pyopenjtalk
  copying pyopenjtalk\version.py -> build\lib.win-amd64-3.9\pyopenjtalk
  copying pyopenjtalk\__init__.py -> build\lib.win-amd64-3.9\pyopenjtalk
  creating build\lib.win-amd64-3.9\pyopenjtalk\htsvoice
  copying pyopenjtalk\htsvoice\LICENSE_mei_normal.htsvoice -> build\lib.win-amd64-3.9\pyopenjtalk\htsvoice
  copying pyopenjtalk\htsvoice\mei_normal.htsvoice -> build\lib.win-amd64-3.9\pyopenjtalk\htsvoice
  copying pyopenjtalk\htsvoice\README.md -> build\lib.win-amd64-3.9\pyopenjtalk\htsvoice
  running build_ext
  cythoning pyopenjtalk\openjtalk.pyx to pyopenjtalk\openjtalk.cpp
  C:\Users\mymc\AppData\Local\Temp\pip-build-env-k2qnpt99\overlay\Lib\site-packages\Cython\Compiler\Main.py:369: FutureWarning: Cython directive 'language_level' not set, using 2 for now (Py2). This will change in a later release! File: C:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\pyopenjtalk\openjtalk\__init__.pxd
    tree = Parsing.p_module(s, pxd, full_module_name)
  cythoning pyopenjtalk\htsengine.pyx to pyopenjtalk\htsengine.cpp
  C:\Users\mymc\AppData\Local\Temp\pip-build-env-k2qnpt99\overlay\Lib\site-packages\Cython\Compiler\Main.py:369: FutureWarning: Cython directive 'language_level' not set, using 2 for now (Py2). This will change in a later release! File: C:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\pyopenjtalk\htsengine\__init__.pxd
    tree = Parsing.p_module(s, pxd, full_module_name)
  building 'pyopenjtalk.openjtalk' extension
  creating build\temp.win-amd64-3.9
  creating build\temp.win-amd64-3.9\Release
  creating build\temp.win-amd64-3.9\Release\lib
  creating build\temp.win-amd64-3.9\Release\lib\open_jtalk
  creating build\temp.win-amd64-3.9\Release\lib\open_jtalk\src
  creating build\temp.win-amd64-3.9\Release\lib\open_jtalk\src\jpcommon
  creating build\temp.win-amd64-3.9\Release\lib\open_jtalk\src\mecab
  creating build\temp.win-amd64-3.9\Release\lib\open_jtalk\src\mecab\src
  creating build\temp.win-amd64-3.9\Release\lib\open_jtalk\src\mecab2njd
  creating build\temp.win-amd64-3.9\Release\lib\open_jtalk\src\njd2jpcommon
  creating build\temp.win-amd64-3.9\Release\lib\open_jtalk\src\njd
  creating build\temp.win-amd64-3.9\Release\lib\open_jtalk\src\njd_set_accent_phrase
  creating build\temp.win-amd64-3.9\Release\lib\open_jtalk\src\njd_set_accent_type
  creating build\temp.win-amd64-3.9\Release\lib\open_jtalk\src\njd_set_digit
  creating build\temp.win-amd64-3.9\Release\lib\open_jtalk\src\njd_set_long_vowel
  creating build\temp.win-amd64-3.9\Release\lib\open_jtalk\src\njd_set_pronunciation
  creating build\temp.win-amd64-3.9\Release\lib\open_jtalk\src\njd_set_unvoiced_vowel
  creating build\temp.win-amd64-3.9\Release\lib\open_jtalk\src\text2mecab
  creating build\temp.win-amd64-3.9\Release\pyopenjtalk
  "C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.37.32822\bin\HostX86\x64\cl.exe" /c /nologo /O2 /W3 /GL /DNDEBUG /MD -DHAVE_CONFIG_H -DDIC_VERSION=102 -DMECAB_DEFAULT_RC=\"dummy\" -DPACKAGE=\"open_jtalk\" -DVERSION=\"1.10\" -DCHARSET_UTF_8 -IC:\Users\mymc\AppData\Local\Temp\pip-build-env-k2qnpt99\overlay\Lib\site-packages\numpy\core\include -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\jpcommon -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\mecab/src -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\mecab2njd -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd2jpcommon -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_accent_phrase -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_accent_type -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_digit -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_long_vowel -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_pronunciation -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_unvoiced_vowel -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\text2mecab -IC:\Users\mymc\Desktop\python\learn_ai\.venv\include -IC:\Users\mymc\.pyenv\pyenv-win\versions\3.9.13\include -IC:\Users\mymc\.pyenv\pyenv-win\versions\3.9.13\include "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.37.32822\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.37.32822\ATLMFC\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Auxiliary\VS\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\um" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\shared" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\winrt" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\cppwinrt" "-IC:\Program Files (x86)\Windows Kits\NETFXSDK\4.8\include\um" /Tclib\open_jtalk\src\jpcommon\jpcommon.c /Fobuild\temp.win-amd64-3.9\Release\lib\open_jtalk\src\jpcommon\jpcommon.obj /source-charset:utf-8 /execution-charset:utf-8
  jpcommon.c
  "C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.37.32822\bin\HostX86\x64\cl.exe" /c /nologo /O2 /W3 /GL /DNDEBUG /MD -DHAVE_CONFIG_H -DDIC_VERSION=102 -DMECAB_DEFAULT_RC=\"dummy\" -DPACKAGE=\"open_jtalk\" -DVERSION=\"1.10\" -DCHARSET_UTF_8 -IC:\Users\mymc\AppData\Local\Temp\pip-build-env-k2qnpt99\overlay\Lib\site-packages\numpy\core\include -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\jpcommon -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\mecab/src -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\mecab2njd -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd2jpcommon -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_accent_phrase -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_accent_type -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_digit -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_long_vowel -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_pronunciation -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_unvoiced_vowel -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\text2mecab -IC:\Users\mymc\Desktop\python\learn_ai\.venv\include -IC:\Users\mymc\.pyenv\pyenv-win\versions\3.9.13\include -IC:\Users\mymc\.pyenv\pyenv-win\versions\3.9.13\include "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.37.32822\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.37.32822\ATLMFC\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Auxiliary\VS\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\um" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\shared" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\winrt" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\cppwinrt" "-IC:\Program Files (x86)\Windows Kits\NETFXSDK\4.8\include\um" /Tclib\open_jtalk\src\jpcommon\jpcommon_label.c /Fobuild\temp.win-amd64-3.9\Release\lib\open_jtalk\src\jpcommon\jpcommon_label.obj /source-charset:utf-8 /execution-charset:utf-8
  jpcommon_label.c
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(115): warning C4996: 'strdup': The POSIX name for this item is deprecated. Instead, use the ISO C and C++ conformant name: _strdup. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(128): warning C4996: 'strdup': The POSIX name for this item is deprecated. Instead, use the ISO C and C++ conformant name: _strdup. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(148): warning C4996: 'strdup': The POSIX name for this item is deprecated. Instead, use the ISO C and C++ conformant name: _strdup. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(168): warning C4996: 'strdup': The POSIX name for this item is deprecated. Instead, use the ISO C and C++ conformant name: _strdup. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(181): warning C4996: 'strdup': The POSIX name for this item is deprecated. Instead, use the ISO C and C++ conformant name: _strdup. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(194): warning C4996: 'strdup': The POSIX name for this item is deprecated. Instead, use the ISO C and C++ conformant name: _strdup. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(207): warning C4996: 'strdup': The POSIX name for this item is deprecated. Instead, use the ISO C and C++ conformant name: _strdup. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(231): warning C4996: 'strdup': The POSIX name for this item is deprecated. Instead, use the ISO C and C++ conformant name: _strdup. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(459): warning C4996: 'strdup': The POSIX name for this item is deprecated. Instead, use the ISO C and C++ conformant name: _strdup. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(462): warning C4996: 'strdup': The POSIX name for this item is deprecated. Instead, use the ISO C and C++ conformant name: _strdup. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(701): warning C4996: 'strcat': This function or variable may be unsafe. Consider using strcat_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(715): warning C4996: 'strcat': This function or variable may be unsafe. Consider using strcat_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(721): warning C4996: 'strcat': This function or variable may be unsafe. Consider using strcat_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(735): warning C4996: 'strcat': This function or variable may be unsafe. Consider using strcat_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(750): warning C4996: 'strcat': This function or variable may be unsafe. Consider using strcat_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(757): warning C4996: 'strcat': This function or variable may be unsafe. Consider using strcat_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(776): warning C4996: 'strcat': This function or variable may be unsafe. Consider using strcat_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(791): warning C4996: 'strcat': This function or variable may be unsafe. Consider using strcat_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(798): warning C4996: 'strcat': This function or variable may be unsafe. Consider using strcat_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(811): warning C4996: 'strcat': This function or variable may be unsafe. Consider using strcat_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(834): warning C4996: 'strcat': This function or variable may be unsafe. Consider using strcat_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(847): warning C4996: 'strcat': This function or variable may be unsafe. Consider using strcat_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(853): warning C4996: 'strcat': This function or variable may be unsafe. Consider using strcat_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(688): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(692): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(698): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(712): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(714): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(718): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(720): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(732): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(734): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(744): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(746): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(752): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(754): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(764): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(768): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(785): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(787): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(793): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(795): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(807): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(809): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(818): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(823): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(843): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(845): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_label.c(849): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  "C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.37.32822\bin\HostX86\x64\cl.exe" /c /nologo /O2 /W3 /GL /DNDEBUG /MD -DHAVE_CONFIG_H -DDIC_VERSION=102 -DMECAB_DEFAULT_RC=\"dummy\" -DPACKAGE=\"open_jtalk\" -DVERSION=\"1.10\" -DCHARSET_UTF_8 -IC:\Users\mymc\AppData\Local\Temp\pip-build-env-k2qnpt99\overlay\Lib\site-packages\numpy\core\include -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\jpcommon -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\mecab/src -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\mecab2njd -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd2jpcommon -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_accent_phrase -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_accent_type -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_digit -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_long_vowel -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_pronunciation -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_unvoiced_vowel -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\text2mecab -IC:\Users\mymc\Desktop\python\learn_ai\.venv\include -IC:\Users\mymc\.pyenv\pyenv-win\versions\3.9.13\include -IC:\Users\mymc\.pyenv\pyenv-win\versions\3.9.13\include "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.37.32822\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.37.32822\ATLMFC\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Auxiliary\VS\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\um" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\shared" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\winrt" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\cppwinrt" "-IC:\Program Files (x86)\Windows Kits\NETFXSDK\4.8\include\um" /Tclib\open_jtalk\src\jpcommon\jpcommon_node.c /Fobuild\temp.win-amd64-3.9\Release\lib\open_jtalk\src\jpcommon\jpcommon_node.obj /source-charset:utf-8 /execution-charset:utf-8
  jpcommon_node.c
  lib\open_jtalk\src\jpcommon\jpcommon_node.c(76): warning C4996: 'strdup': The POSIX name for this item is deprecated. Instead, use the ISO C and C++ conformant name: _strdup. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_node.c(83): warning C4996: 'strdup': The POSIX name for this item is deprecated. Instead, use the ISO C and C++ conformant name: _strdup. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_node.c(90): warning C4996: 'strdup': The POSIX name for this item is deprecated. Instead, use the ISO C and C++ conformant name: _strdup. See online help for details.
  lib\open_jtalk\src\jpcommon\jpcommon_node.c(97): warning C4996: 'strdup': The POSIX name for this item is deprecated. Instead, use the ISO C and C++ conformant name: _strdup. See online help for details.
  "C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.37.32822\bin\HostX86\x64\cl.exe" /c /nologo /O2 /W3 /GL /DNDEBUG /MD -DHAVE_CONFIG_H -DDIC_VERSION=102 -DMECAB_DEFAULT_RC=\"dummy\" -DPACKAGE=\"open_jtalk\" -DVERSION=\"1.10\" -DCHARSET_UTF_8 -IC:\Users\mymc\AppData\Local\Temp\pip-build-env-k2qnpt99\overlay\Lib\site-packages\numpy\core\include -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\jpcommon -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\mecab/src -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\mecab2njd -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd2jpcommon -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_accent_phrase -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_accent_type -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_digit -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_long_vowel -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_pronunciation -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_unvoiced_vowel -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\text2mecab -IC:\Users\mymc\Desktop\python\learn_ai\.venv\include -IC:\Users\mymc\.pyenv\pyenv-win\versions\3.9.13\include -IC:\Users\mymc\.pyenv\pyenv-win\versions\3.9.13\include "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.37.32822\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.37.32822\ATLMFC\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Auxiliary\VS\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\um" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\shared" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\winrt" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\cppwinrt" "-IC:\Program Files (x86)\Windows Kits\NETFXSDK\4.8\include\um" /EHsc /Tplib\open_jtalk\src\mecab/src\char_property.cpp /Fobuild\temp.win-amd64-3.9\Release\lib\open_jtalk\src\mecab/src\char_property.obj /source-charset:utf-8 /execution-charset:utf-8
  char_property.cpp
  C:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\mecab\src\utils.h(47): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  C:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\mecab\src\scoped_ptr.h(84): warning C4996: 'strcpy': This function or variable may be unsafe. Consider using strcpy_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  C:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\mecab\src\scoped_ptr.h(90): warning C4996: 'strcpy': This function or variable may be unsafe. Consider using strcpy_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  lib\open_jtalk\src\mecab/src\char_property.cpp(269): warning C4996: 'strncpy': This function or variable may be unsafe. Consider using strncpy_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  "C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.37.32822\bin\HostX86\x64\cl.exe" /c /nologo /O2 /W3 /GL /DNDEBUG /MD -DHAVE_CONFIG_H -DDIC_VERSION=102 -DMECAB_DEFAULT_RC=\"dummy\" -DPACKAGE=\"open_jtalk\" -DVERSION=\"1.10\" -DCHARSET_UTF_8 -IC:\Users\mymc\AppData\Local\Temp\pip-build-env-k2qnpt99\overlay\Lib\site-packages\numpy\core\include -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\jpcommon -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\mecab/src -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\mecab2njd -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd2jpcommon -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_accent_phrase -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_accent_type -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_digit -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_long_vowel -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_pronunciation -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_unvoiced_vowel -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\text2mecab -IC:\Users\mymc\Desktop\python\learn_ai\.venv\include -IC:\Users\mymc\.pyenv\pyenv-win\versions\3.9.13\include -IC:\Users\mymc\.pyenv\pyenv-win\versions\3.9.13\include "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.37.32822\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.37.32822\ATLMFC\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Auxiliary\VS\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\um" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\shared" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\winrt" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\cppwinrt" "-IC:\Program Files (x86)\Windows Kits\NETFXSDK\4.8\include\um" /EHsc /Tplib\open_jtalk\src\mecab/src\connector.cpp /Fobuild\temp.win-amd64-3.9\Release\lib\open_jtalk\src\mecab/src\connector.obj /source-charset:utf-8 /execution-charset:utf-8
  connector.cpp
  C:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\mecab\src\utils.h(47): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  C:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\mecab\src\scoped_ptr.h(84): warning C4996: 'strcpy': This function or variable may be unsafe. Consider using strcpy_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  C:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\mecab\src\scoped_ptr.h(90): warning C4996: 'strcpy': This function or variable may be unsafe. Consider using strcpy_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  C:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\mecab\src\connector.h(36): warning C4267: '=': 'size_t' から 'unsigned short' に変換しました。データが失われているかもしれません。
  C:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\mecab\src\connector.h(37): warning C4267: '=': 'size_t' から 'unsigned short' に変換しました。データが失われているかもしれません。
  "C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.37.32822\bin\HostX86\x64\cl.exe" /c /nologo /O2 /W3 /GL /DNDEBUG /MD -DHAVE_CONFIG_H -DDIC_VERSION=102 -DMECAB_DEFAULT_RC=\"dummy\" -DPACKAGE=\"open_jtalk\" -DVERSION=\"1.10\" -DCHARSET_UTF_8 -IC:\Users\mymc\AppData\Local\Temp\pip-build-env-k2qnpt99\overlay\Lib\site-packages\numpy\core\include -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\jpcommon -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\mecab/src -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\mecab2njd -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd2jpcommon -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_accent_phrase -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_accent_type -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_digit -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_long_vowel -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_pronunciation -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_unvoiced_vowel -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\text2mecab -IC:\Users\mymc\Desktop\python\learn_ai\.venv\include -IC:\Users\mymc\.pyenv\pyenv-win\versions\3.9.13\include -IC:\Users\mymc\.pyenv\pyenv-win\versions\3.9.13\include "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.37.32822\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.37.32822\ATLMFC\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Auxiliary\VS\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\um" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\shared" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\winrt" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\cppwinrt" "-IC:\Program Files (x86)\Windows Kits\NETFXSDK\4.8\include\um" /EHsc /Tplib\open_jtalk\src\mecab/src\context_id.cpp /Fobuild\temp.win-amd64-3.9\Release\lib\open_jtalk\src\mecab/src\context_id.obj /source-charset:utf-8 /execution-charset:utf-8
  context_id.cpp
  C:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\mecab\src\utils.h(47): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  "C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.37.32822\bin\HostX86\x64\cl.exe" /c /nologo /O2 /W3 /GL /DNDEBUG /MD -DHAVE_CONFIG_H -DDIC_VERSION=102 -DMECAB_DEFAULT_RC=\"dummy\" -DPACKAGE=\"open_jtalk\" -DVERSION=\"1.10\" -DCHARSET_UTF_8 -IC:\Users\mymc\AppData\Local\Temp\pip-build-env-k2qnpt99\overlay\Lib\site-packages\numpy\core\include -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\jpcommon -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\mecab/src -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\mecab2njd -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd2jpcommon -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_accent_phrase -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_accent_type -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_digit -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_long_vowel -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_pronunciation -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\njd_set_unvoiced_vowel -IC:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\text2mecab -IC:\Users\mymc\Desktop\python\learn_ai\.venv\include -IC:\Users\mymc\.pyenv\pyenv-win\versions\3.9.13\include -IC:\Users\mymc\.pyenv\pyenv-win\versions\3.9.13\include "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.37.32822\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.37.32822\ATLMFC\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Auxiliary\VS\include" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.22621.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\um" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\shared" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\winrt" "-IC:\Program Files (x86)\Windows Kits\10\\include\10.0.22621.0\\cppwinrt" "-IC:\Program Files (x86)\Windows Kits\NETFXSDK\4.8\include\um" /EHsc /Tplib\open_jtalk\src\mecab/src\dictionary.cpp /Fobuild\temp.win-amd64-3.9\Release\lib\open_jtalk\src\mecab/src\dictionary.obj /source-charset:utf-8 /execution-charset:utf-8
  dictionary.cpp
  C:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\mecab\src\utils.h(47): warning C4996: 'sprintf': This function or variable may be unsafe. Consider using sprintf_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  C:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\mecab\src\scoped_ptr.h(84): warning C4996: 'strcpy': This function or variable may be unsafe. Consider using strcpy_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  C:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\mecab\src\scoped_ptr.h(90): warning C4996: 'strcpy': This function or variable may be unsafe. Consider using strcpy_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  C:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\mecab\src\connector.h(36): warning C4267: '=': 'size_t' から 'unsigned short' に変換しました。データが失われているかもしれません。
  C:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc\lib\open_jtalk\src\mecab\src\connector.h(37): warning C4267: '=': 'size_t' から 'unsigned short' に変換しました。データが失われているかもしれません。
  lib\open_jtalk\src\mecab/src\dictionary.cpp(68): error C2143: 構文エラー: ',' が '<' の前にありません。
  lib\open_jtalk\src\mecab/src\dictionary.cpp(73): note: コンパイル対象の クラス テンプレート インスタンス化 'MeCab::`anonymous-namespace'::pair_1st_cmp<T1,T2>' のリファレンスを確認してください    
  lib\open_jtalk\src\mecab/src\dictionary.cpp(418): warning C4267: '=': 'size_t' から 'unsigned short' に変換しました。データが失われているかもしれません。
  lib\open_jtalk\src\mecab/src\dictionary.cpp(419): warning C4267: '=': 'size_t' から 'unsigned short' に変換しました。データが失われているかもしれません。
  lib\open_jtalk\src\mecab/src\dictionary.cpp(444): warning C4267: '=': 'size_t' から 'unsigned int' に変換しました。データが失われているかもしれません。
  lib\open_jtalk\src\mecab/src\dictionary.cpp(68): error C2039: 'binary_function': 'std' のメンバーではありません
  C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\VC\Tools\MSVC\14.37.32822\include\map(25): note: 'std' の宣言を確認してください
  lib\open_jtalk\src\mecab/src\dictionary.cpp(466): note: コンパイル対象の クラス テンプレート インスタンス化 'MeCab::`anonymous-namespace'::pair_1st_cmp<std::string,MeCab::Token *>' のリファレンスを確認してください
  lib\open_jtalk\src\mecab/src\dictionary.cpp(68): error C2504: 'binary_function': 定義されていない基底クラスが宣言されています。
  lib\open_jtalk\src\mecab/src\dictionary.cpp(479): warning C4267: '引数': 'size_t' から '_Ty' に変換しました。データが失われているかもしれません。
          with
          [
              _Ty=int
          ]
  lib\open_jtalk\src\mecab/src\dictionary.cpp(489): warning C4267: '引数': 'size_t' から '_Ty' に変換しました。データが失われているかもしれません。
          with
          [
              _Ty=int
          ]
  lib\open_jtalk\src\mecab/src\dictionary.cpp(515): warning C4267: '初期化中': 'size_t' から 'unsigned int' に変換しました。データが失われているかもしれません。
  lib\open_jtalk\src\mecab/src\dictionary.cpp(516): warning C4267: '初期化中': 'size_t' から 'unsigned int' に変換しました。データが失われているかもしれません。
  lib\open_jtalk\src\mecab/src\dictionary.cpp(517): warning C4267: '初期化中': 'size_t' から 'unsigned int' に変換しました。データが失われているかもしれません。
  lib\open_jtalk\src\mecab/src\dictionary.cpp(518): warning C4267: '初期化中': 'size_t' から 'unsigned int' に変換しました。データが失われているかもしれません。
  lib\open_jtalk\src\mecab/src\dictionary.cpp(519): warning C4267: '初期化中': 'size_t' から 'unsigned int' に変換しました。データが失われているかもしれません。
  lib\open_jtalk\src\mecab/src\dictionary.cpp(524): warning C4996: 'strncpy': This function or variable may be unsafe. Consider using strncpy_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
  error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2022\\BuildTools\\VC\\Tools\\MSVC\\14.37.32822\\bin\\HostX86\\x64\\cl.exe' failed with exit code 2
  error: subprocess-exited-with-error

  × Building wheel for pyopenjtalk (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> See above for output.

  note: This error originates from a subprocess, and is likely not a problem with pip.
  full command: 'C:\Users\mymc\Desktop\python\learn_ai\.venv\Scripts\python.exe' 'C:\Users\mymc\Desktop\python\learn_ai\.venv\lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py' build_wheel 'C:\Users\mymc\AppData\Local\Temp\tmpv3930zfz'
  cwd: C:\Users\mymc\AppData\Local\Temp\pip-install-w5__eo7i\pyopenjtalk_4ec3f9e98bf54ec589146958d412ddfc
  Building wheel for pyopenjtalk (pyproject.toml) ... error
  ERROR: Failed building wheel for pyopenjtalk
Failed to build pyopenjtalk
ERROR: Could not build wheels for pyopenjtalk, which is required to install pyproject.toml-based projects
```
</details>

### 再現手順

1. MSVC 14.37に更新する（ダウングレードできないので注意してください。）
2. `pyopenjtalk`を再インストールする

### 期待動作

正常にインストールできる。

## VOICEVOXのバージョン

0.?.0

<!-- "ヘルプ" → "アップデート情報" で確認できます -->

## OSの種類/ディストリ/バージョン

<!-- チェックするには [ ] を [x] に変更してください -->

- [x] Windows
- [ ] macOS
- [ ] Linux

<!--
なるべく詳しく書いてください 記述例:
*   Windows 10 Pro 64bit (10.0.10586)
*   macOS Sierra
*   Linux fedora 23 64bit
*   Others
-->

## その他

<!-- 関連して何か気がついたこと、気になることがあればココに書いてください -->

--- a/src/mecab/src/dictionary.cpp
+++ b/src/mecab/src/dictionary.cpp
@@ -5,6 +5,7 @@
 //  Copyright(C) 2004-2006 Nippon Telegraph and Telephone Corporation
 #include <fstream>
 #include <climits>
+#include <functional>
 #include "connector.h"
 #include "context_id.h"
 #include "char_property.h"

